packages/taquito-contracts-library/src/read-provider-wrapper.ts

```diff
@@ -46,6 +46,7 @@ export class ReadWrapperContractsLibrary implements TzReadProvider {
     hard_gas_limit_per_block: BigNumber;
     hard_storage_limit_per_operation: BigNumber;
     cost_per_byte: BigNumber;
+    smart_rollup_origination_size: number;
```

Comment:
is this now a proto constant?

Reply:
yes but based on the estimate its a bit more complicated to try to estimate it ourself and its held in the estimate return within `result.size`

Reply:
do you want this constant removed if it currently isnt in use?